arm-web/2015-08-01/swagger/AppServiceCertificateOrders.json

```diff
@@ -713,33 +720,20 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/CertificateEmail"
-              }
-            }
+          "204": {
```

Comment:
What happens if the server fails to verify, can you set a default response here?